src/accountsdb/buffer_pool.zig

```diff
@@ -196,12 +244,11 @@ pub const BufferPool = struct {
         size: FrameOffset,
     ) error{CannotOverwriteAliveInfo}!void {
```

Comment:
theres a `    /// TODO: atomics` above this method which i dont think we need?

Reply:
This is gone / not sure what commit I removed it in